src/components/parent-form/ParticipantInfoPage.tsx

```diff
@@ -7,36 +7,49 @@ import {
     Input,
     HStack,
     Select,
+    Button,
+    FormErrorMessage,
 } from "@chakra-ui/react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import moment from "moment";
+import colourTheme from "@styles/colours";
+import { PostalCodeField } from "@components/formFields/PostalCodeField";
+import { TextField } from "@components/formFields/TextField";
+import { ProvinceField } from "@components/formFields/ProvinceField";
+import { CheckBoxField } from "@components/formFields/CheckBoxField";
+import { DateField } from "@components/formFields/DateField";
 
 import { province } from "@models/User";
+import validator from "validator";
```

Comment:
There seems to be some unused imports. Please do an organize import on your files!